compiler/rustc_llvm/llvm-wrapper/CoverageMappingWrapper.cpp

```diff
@@ -1,9 +1,11 @@
 #include "LLVMWrapper.h"
+#include "llvm/ADT/ArrayRef.h"
 #include "llvm/ProfileData/Coverage/CoverageMapping.h"
 #include "llvm/ProfileData/Coverage/CoverageMappingWriter.h"
 #include "llvm/ProfileData/InstrProf.h"
-#include "llvm/ADT/ArrayRef.h"
 
+#include <array>
+#include <cstdint>
```

Comment:
Are these necessary for anything?

Reply:
Clearly not. :grimacing:
These lines may be inserted by clangd automatically. I should check this file carefully again (not just for format). Thanks for your carefulness！